tests/resources/errors.tsv

```diff
@@ -1,7 +1,7 @@
 table	cell	rule ID	rule	level	message	suggestion
-external	E7	field:8		ERROR	"'Degue virus' must meet one of: blank or under(external.Parent, ""material entity"")"	
-external	D11	field:7		ERROR	'process' from external.Parent must exist in external.Label	
-exposure	E5	rule:7	"When the exposure process is ""occurrence of disease"" then the disease reported must be provided."	ERROR	"because 'occurrence of disease' is 'under(external.Parent, ""occurrence of disease"")', value cannot be blank"	
-exposure	B6	rule:4	The exposure material ID must match the exposure material reported.	ERROR	because 'Dengue virus' is 'not blank', 'NCBITaxon:11103' must be 'NCBITaxon:12637'	
-external	A16	field:5	non-distinct value(s)	ERROR	'ONTIE:0003317' must be distinct with value(s) at: external:A15	
+external	E7	field:8		ERROR	'Degue virus' must meet one of: blank, under	
```

Comment:
We lost some useful information here.

Reply:
If we want to keep the full unparsed function, we will have to add the "unparsed_condition" arg to a lot of functions. It is useful, but I don't see another good way to do it. The conditions can get messy when they're nested, so it might be hard to write something to convert a parsed function back to an unparsed function.